community/mei-board/protocols/2025-08-18_Virtual-Board-Meeting.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: "2025-08-18 Virtual MEI Board Meeting"
+---
+
+# 2025 August 18 Virtual MEI Board Meeting
+
+Present: Anna Kijas, Martha Thomae Elias, Maristella Feustle, Benjamin Bohl, David Weigl, Anna Plaksin, Johannes Kepper, Laurent Pugin
+Absent: Stefan Münnich
+
+Minutes: Anna Kijas
+
+## Agenda
+
+Updates for MEC 2026 (Johannes/Anna P.)
+- Met with Tokyo host (Eita) to discuss conference planning and expectations
+- Discussed using Conftool, conference management system, for proposal submissions and registration
+
+
+Discussion
+- Draft MOU for conference hosts
+    - Document would set expectations for conference hosts
+    - Board members provided feedback and finalized document will be shared when ready
+
+- Draft invitation for MEI Institutional Sponsors
+    - Invitation would be sent to ally organizations/institutions requesting tiered sponsorship opportunities to help support MEI activities, such as bursaries and conferences
+    - Board members provided feedback and finalized document will be shared when ready
+- Opportunity to bring a second MEI sponsor on board who would oversee MEI’s financial accounts
```

Comment:
“`suggestion
- Opportunity to bring a second MEI host institution on board, who would oversee MEI’s financial accounts
```